bolt-portal/src/pages/TestExecutions/components/CPUWarningBadge/CPUWarningBadge.style.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2022 Acaisoft
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { makeStyles } from '@material-ui/core'
+
+export default makeStyles(({ palette, spacing }) => ({
+  icon: {
+    margin: spacing(1.2),
+    fontSize: '2.4rem',
+    color: palette.text.warning
+  },
+  text: {
+    margin: spacing(2),
+    fontSize: '0.9rem',
+    color: palette.text.primary,
+    whiteSpace: 'pre-wrap'
+  },
+  wrapper: {
+    padding: spacing(0.5, 1),
+    borderRadius: spacing(0.5),
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  title: {
```

Comment:
The `title` style is defined but not used anywhere in the component. Remove unused styles.